lib/server.js

```diff
@@ -0,0 +1,40 @@
+let { loadExtension } = require("./util");
+
+exports.static = function(config, webroot) {
+	let donny = loadExtension("donny",
+			"you need to install donny to use --serve");
+	let [bind, port] = parse(config);
+
+	donny({
+		port,
+		bind,
+		webroot
+	}).then(() => {
+		console.error(`Serving '${webroot}' on port ${port}`);
```

Comment:
Emitting just the port isn't particularly user-friendly; we want a proper URI to copy or click on - so `http://${bind}:${port}`?

(Does live-serve emit such a message automatically?)

Reply:
Agreed, feel free to adjust it! And yes, live-serve posts the message on its own.

Reply:
```
console.error(`serving ${repr(webroot)} at http://${host}:${port}`);
```
✅